net.c

```diff
@@ -457,7 +457,7 @@ static int _redisContextConnectTcp(redisContext *c, const char *addr, int port,
             }
 
             for (b = bservinfo; b != NULL; b = b->ai_next) {
-                if (bind(s,b->ai_addr,b->ai_addrlen) != -1) {
+                if (bind(s,b->ai_addr,(socklen_t)b->ai_addrlen) != -1) {
```

Comment:
Windows has ai_addren as `size_t` rather than `socklen_t`.  explicitly cast it to silence warning.